File: typescript/26-clamp-number.ts
// Clamp de número — Fácil · Números
// Limita num al rango inclusivo [min, max].
// Fuente: https://coding-challenges.dev/problems/clamp-de-numero

function clampNumber(num: number, min: number, max: number): number {
  return Math.max(min, Math.min(num, max));
}

// No modificar: necesario para evaluar el resultado.
export { clampNumber };

File: python/26_clamp_number.py
"""Clamp de número — Fácil · Números

Limita num al rango inclusivo [min_val, max_val].

Fuente: https://coding-challenges.dev/problems/python-clamp-de-numero
"""


def clamp_number(num: int, min_val: int, max_val: int) -> int:
    return max(min_val, min(num, max_val))
